lnwallet/channel_test.go

```diff
@@ -8014,6 +8014,10 @@ func TestChannelMaxFeeRate(t *testing.T) {
 				"allocation of %v, got %v", expFeeRate,
 				maxAlloc, maxFeeRate)
 		}
+
+		if err := c.validateFeeRate(maxFeeRate); err != nil {
+			t.Fatalf("fee rate validation failed: %v", err)
+		}
```

Comment:
note that before this commit, adding this validation check here would fail the test

Reply:
ok i have tried this. See `propertyTest` below.